app/views/articles/show.html.erb

```diff
@@ -85,16 +133,14 @@
     <% if @article.video_state == "PROGRESSING" %>
       <h1 style="text-align:center;">⏳ Video Transcoding In Progress ⏳</h1>
     <% end %>
-    <article itemscope itemtype="http://schema.org/Article" itemprop="mainEntity">
-      <meta itemprop="url" content="<%= article_url(@article) %>">
-      <meta itemprop="image" content="<%= article_social_image_url(@article) %>">
-      <div itemprop="publisher" itemscope itemtype="https://schema.org/Organization">
-        <div itemprop="logo" itemscope itemtype="https://schema.org/ImageObject">
-          <meta itemprop="url" content="<%= cloudinary(SiteConfig.logo_png, 192, "png") %>">
-          <meta itemprop="width" content="192">
-          <meta itemprop="height" content="192">
+    <article>
+      <% article_url(@article) %>
+      <% article_social_image_url(@article) %>
+      <div>
+        <div>
+          <% cloudinary(SiteConfig.logo_png, 192, "png") %>
         </div>
-        <meta itemprop="name" content="<%= community_qualified_name %>">
+        <% community_qualified_name %>
```

Comment:
```suggestion
        <%= community_qualified_name %>
```
should this be invoked here? looking at the diff, it seems like we used to do this: `<%= community_qualified_name %>">`, so i'm confused why we're not doing `<%= community_qualified_name %>`

Reply:
Good question! I actually meant to add this to my list of concerns. When I remove the `<meta>` tag that was wrapped around the `erb`, the code within the `erb` renders on the webpage as text, as shown in the before and after pictures below. 

Two possible solutions that I've found so far are to either 1.) not invoke the `erb` or 2.) wrap the `erb` in `<a>` tags in place of the `<meta>` tags. I don't think that either of the two solutions is great and would like more opinions on this or possible solutions for it. 😄 

With `<meta>` tags removed and without invoking the `erb`:
![Screen Shot 2020-04-09 at 1 15 44 PM](https://user-images.githubusercontent.com/32834804/78932627-1d4b9680-7a65-11ea-9599-7677625d5d13.png)


With the `<meta>` tags removed, while still invoking the `erb`:
![Screen Shot 2020-04-09 at 1 16 16 PM](https://user-images.githubusercontent.com/32834804/78932815-6bf93080-7a65-11ea-9db1-f35d453ea976.png)

Reply:
Ah, when I first looked at this diff, I didn't actually catch that this was inside of the `<meta>`. My next question would be: is removing the `<meta>` tag intentional??

If it _is_ and we _do_ want to remove it, then you should just remove the `<% community_qualified_name %>`; you aren't invoking it, so it's on the template but isn't being used at all.

`<% community_qualified_name %>` is just executing the variable but isn't rendering it, so it's wasted effort because that variable is thrown away.

Reply:
> When I remove the <meta> tag that was wrapped around the erb, the code within the erb renders on the webpage as text, as shown in the before and after pictures below.

This is expected, because you're just actually just invoking what's inside the erb, you're actually rendering it! :) In one situation you have an expression, in another situation you have a scriptlet, as described [here](https://www.stuartellis.name/articles/erb).

Reply:
OH, I see; that makes sense! To answer your question, yes, removing the `<meta>` tag was intentional, but I was hesitant to remove the `erb` from the template after removing the `<meta>` tags because I wasn't sure if I was removing too much, but now that you mention it, it seems like it's actually just dead code if I leave the `erb` sans `<meta>` tags in there.